templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h1>Delete manufacturer</h1>
+  <p>You really want to delete the manufacturer: {{ manufacturer }}</p>
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes" class="btn btn-danger">
```

Comment:
Add a screenshot of this page together with the `Cancel` button that should lead to a previous page according [to the checklist](https://github.com/Shum430/py-taxi-service-forms/blob/develop/checklist.md#code-style).